src/payment/strategies/klarna/klarna-payment-strategy.ts

```diff
@@ -114,6 +144,35 @@ export default class KlarnaPaymentStrategy implements PaymentStrategy {
             }));
     }
 
+    private _loadPaymentsWidget(options: PaymentInitializeOptions): Promise<KlarnaLoadResponse> {
+        if (!options.klarnav2) {
+            throw new InvalidArgumentError('Unable to load widget because "options.klarna" argument is not provided.');
+        }
+
+        const { methodId, klarnav2: { container, payment_method_category, onLoad } } = options;
+        const state = this._store.getState();
+
+        return new Promise<KlarnaLoadResponse>(resolve => {
+            const paymentMethod = state.paymentMethods.getPaymentMethod(methodId);
```

Comment:
You can use the new method I've added recently - `getPaymentMethodOrThrow` - so you don't have to do the check you're doing below.